buildSrc/src/main/groovy/io/micronaut/guides/OpenAPIGenerator.groovy

```diff
@@ -0,0 +1,70 @@
+package io.micronaut.guides
+
+import io.micronaut.starter.api.TestFramework
+import io.micronaut.starter.options.BuildTool
+import io.micronaut.starter.options.Language
+import org.gradle.api.GradleException
+import org.openapitools.codegen.ClientOptInput
+import org.openapitools.codegen.DefaultGenerator
+import org.openapitools.codegen.Generator
+import org.openapitools.codegen.GeneratorNotFoundException
+import org.openapitools.codegen.config.CodegenConfigurator
+
+import static io.micronaut.starter.api.TestFramework.JUNIT
+import static io.micronaut.starter.api.TestFramework.SPOCK
+import static io.micronaut.starter.options.BuildTool.GRADLE
+import static io.micronaut.starter.options.BuildTool.MAVEN
+
+@Singleton
```

Comment:
No need to annotate this as `@Singleton`.

Reply:
resolved via https://github.com/micronaut-projects/micronaut-guides/pull/728/commits/b34e78642671134c75b7cab67d372e8ac40ccc21